examples/vts-react-test/package-lock.json

```diff
@@ -14,10 +14,19 @@
         "react": "^18.2.0",
         "react-dom": "^18.2.0",
         "react-scripts": "5.0.1",
-        "vtubestudio": "^3.0.0-beta.4",
+        "vtubestudio": "^3.0.0",
```

Comment:
I tested my changes in the react example, which is why the package-lock.json was updated